src/lib.rs

```diff
@@ -259,13 +259,22 @@ impl<K: EnrKey> Enr<K> {
 
     /// Reads a custom key from the record if it exists.
     pub fn get(&self, key: impl AsRef<[u8]>) -> Option<&[u8]> {
+        // It's ok to decode any valid RLP integer as data
```

Comment:
I think you meant this instead?
```suggestion
        // It's ok to decode any valid RLP value as data
```